site/_layouts/doc.liquid

```diff
@@ -221,9 +223,15 @@
                     <li {% if page.url == "/docs/v1/tech/lambdas/ldap-connector-reconcile/" %}class="active"{% endif %}><a href="/docs/v1/tech/lambdas/ldap-connector-reconcile/">LDAP Connector Reconcile</a></li>
                     <li {% if page.url == "/docs/v1/tech/lambdas/linkedin-reconcile/" %}class="active"{% endif %}><a href="/docs/v1/tech/lambdas/linkedin-reconcile/">LinkedIn Reconcile</a></li>
                     <li {% if page.url == "/docs/v1/tech/lambdas/openid-connect-response-reconcile/" %}class="active"{% endif %}><a href="/docs/v1/tech/lambdas/openid-connect-response-reconcile/">OpenID Connect Reconcile</a></li>
+                    <li {% if page.url == "/docs/v1/tech/lambdas/nintendo-reconcile/" %}class="active"{% endif %}><a href="/docs/v1/tech/lambdas/nintendo-reconcile/">Nintendo Reconcile</a></li>
```

Comment:
Do we want to leave this one commented out until we deliver the Nintendo IdP?

Reply:
Good idea!  Just didn't want to forget about my favorite console company (duck hunt FTW!).  Will comment out for now.